bin/gftools-builder.py

```diff
@@ -72,12 +80,33 @@
     builder.config["logLevel"] = "DEBUG"
 
 if args.dump_config:
-    import sys
     import yaml
 
     with open(args.dump_config, "w") as fp:
         config= {k: v for (k, v) in builder.config.items() if v is not None}
         fp.write(yaml.dump(config, Dumper=yaml.SafeDumper))
     sys.exit()
 
+try:
+    config_yaml = load(unprocessed_yaml, schema)
+except YAMLError as error:
+    print(error)
```

Comment:
I like you've retained the spell checking. This may be too verbose for power users/devs but for font designers etc it is perfect.

Instead of calling print, we should just call `YAMLError` just once e.g:

```
except YAMLError as error:
    raise YAMLError(
        f"'\nERROR ' + '*'*64 + '\n'
        f"A key in the configuration file, typically ``config.yaml``, is likely misspelled."
        f"\nError caused by key: {bad_key}"
        ...
    )

Reply:
Thanks. This is done in commit 9463326.